.ci.yaml

```diff
@@ -4178,11 +4178,3 @@ targets:
         ["devicelab", "android", "windows"]
       task_name: windows_chrome_dev_mode
     scheduler: luci
-
-  - name: google_test
```

Comment:
Could you remove the ownership entry as well: https://github.com/flutter/flutter/blob/master/TESTOWNERS#L260